scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -359,7 +358,7 @@
                                                       (map (fn [[k v]]
                                                              [(keyword (name (keyword k))) v]))
                                                       (into {}))
-                                              utilization (get-mesos-utilization mesos-master)
+                                              utilization (get-mesos-utilization mesos-master-hosts)
```

Comment:
What is the purpose of this? Is it just so that the rebalancer only runs when the utilization is above some minimum threshold?